### PR TITLE
Move system upgrade to system info page

### PR DESCRIPTION
**Pull request by gregaubert**
@stas-vilchik to test I change "a bit" the WS in `system.ts` : 
```js
export function getSystemUpgrades(): Promise<{
  upgrades: SystemUpgrade[];
  updateCenterRefresh: string;
}> {
  return getJSON('/api/system/upgrades').then(() => ({
    updateCenterRefresh: '',
    upgrades: [
      {
        version: '5.6.7',
        description: 'Version 5.6.7 description',
        releaseDate: '2017-03-01',
        changeLogUrl: 'changelogurl',
        downloadUrl: 'downloadurl',
        plugins: {}
      },
      {
        version: '5.6.5',
        description: 'Version 5.6.5 description',
        releaseDate: '2017-03-01',
        changeLogUrl: 'changelogurl',
        downloadUrl: 'downloadurl',
        plugins: {}
      },
      {
        version: '6.3',
        description: 'Version 6.3 description',
        releaseDate: '2017-05-02',
        changeLogUrl: 'changelogurl',
        downloadUrl: 'downloadurl',
        plugins: {}
      },
      {
        version: '5.6.6',
        description: 'Version 5.6.6 description',
        releaseDate: '2017-04-02',
        changeLogUrl: 'changelogurl',
        downloadUrl: 'downloadurl',
        plugins: {}
      },
      {
        version: '6.4',
        description: 'Version 6.4 description',
        releaseDate: '2017-06-02',
        changeLogUrl: 'changelogurl',
        downloadUrl: 'downloadurl',
        plugins: {}
      }
    ]
  }));
}
```

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/system/styles.css [hunk 1]
```diff
@@ -61,3 +61,19 @@
...
+
+.system-upgrade-version ~ .system-upgrade-version {
+  margin-top: 16px;
+}
+
+.system-upgrade-intermediate {
+  padding: 6px 10px;
+}
+
+.system-upgrade-version ~ .system-upgrade-version {
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/system/styles.css]: this is duplicate of line 69

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 2]
```diff
@@ -2234,14 +2232,26 @@ system.cluster_log_level.info=Changes apply to all Application nodes but not to
...
 system.log_level.warning=This level has performance impacts, please make sure to get back to INFO level once your investigation is done. Please note that when the server is restarted, logging will revert to the level configured in sonar.properties.
 system.log_level.warning.short=Current logs level has performance impacts, get back to INFO level.
 system.log_level.info=Changes don't apply to Search.
 system.logs_level=Logs level
+system.new_version_available=A new version of SonarQube is available.
+system.release_notes=Release Notes
+system.released_x=Released {0}
 system.restart_server=Restart Server
 system.search_nodes_title=Search Nodes
+system.see_sonarqube_downloads=See all SonarQube Downloads
```
Comment (stas-vilchik) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: capitalize "All"

File: server/sonar-web/src/main/js/apps/system/styles.css (hunk 1 above)
Comment (gregaubert) [reply server/sonar-web/src/main/js/apps/system/styles.css]: :+1:

File: sonar-core/src/main/resources/org/sonar/l10n/core.properties [hunk 3]
```diff
@@ -2096,6 +2096,7 @@ marketplace.i_accept_the=I accept the
 marketplace.commercial_edition=Commercial Edition
 marketplace.terms_and_conditions=Terms and Conditions
 marketplace.editions_unavailable=Explore our Commercial Editions: advanced feature packs brought to you by SonarSource on {url}
+marketplace.release_notes=Release Notes
```
Comment (stas-vilchik) [inline sonar-core/src/main/resources/org/sonar/l10n/core.properties]: 👏

**Pull request merged**